solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -895,3 +963,143 @@ def _name_pfxobs(current_names, forecast, i=1):
         return _name_pfxobs(current_names, new_name, i + 1)
     else:
         return forecast_name
+
+
+def get_forecast_report_issue_times(
```

Comment:
I always question the need for `get_` at the start of function names. Do we really want that here and in the function below?

Reply:
I will remove this. Silly habit from writing literal getters/setters, and this isn't even a class.